packages/bottender/src/slack/SlackConnector.ts

```diff
@@ -96,11 +96,18 @@ export default class SlackConnector
     this._signingSecret = signingSecret || '';
     this._verificationToken = verificationToken || '';
 
-    if (!this._verificationToken) {
-      warning(
-        false,
-        '`verificationToken` is not set. Will bypass Slack event verification.\nPass in `verificationToken` to perform Slack event verification.'
-      );
+    if (!this._signingSecret) {
+      if (!this._verificationToken) {
+        warning(
+          false,
+          '`signingSecret` is not set. Will fallback to `verificationToken`, which is deprecated.'
```

Comment:
should be `signingSecret` and `verificationToken` both not set and bypass checking here